index_V1

```diff
@@ -1,3 +1,28 @@
+<?php
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
+<?php
+	$error = NULL;
+	if($_SERVER["REQUEST_METHOD"] == "POST") {
+		// connect.php (tells where to connect servername, dbaseName, username, password)
+		require "Database_mysqli.php";
+		// username and passwrod sent from form 
+		
+		$search = mysqli_real_escape_string($conn, $_POST['searched']);
+
+		$_SESSION['search-query'] = $search;
+		header("location: query_search.php");
+	}
+?>
+
+
```

Comment:
Above I added the code for the connect page and session start. This make sure the user must Log In before accessing Index.